Sources/_StringProcessing/Regex/DSLTree.swift

```diff
@@ -740,6 +740,32 @@ extension DSLTree {
     @_spi(RegexBuilder)
     public struct Atom {
       internal var ast: AST.Atom
+
+      // FIXME: The below APIs should be removed once the DSL tree has been
+      // migrated to use proper DSL atoms for them.
```

Comment:
What does this entail?

Reply:
Adding them as dedicated cases to `DSLTree.Atom` (or splitting out a new `BuiltinCharacterClass` type perhaps), and then getting the consumer logic working with them. It's probably not too effort, but I'm going to leave it as a follow up for now. I can file an issue to track it.

Reply:
Filed https://github.com/apple/swift-experimental-string-processing/issues/579